packages/components/rollup.config.js

```diff
@@ -16,7 +16,7 @@ const baseConfig = {
   plugins: {
     preVue: [
       alias({
-        resolve: ['.js', '.jsx', '.ts', '.tsx', '.vue', '.svg'],
+        resolve: ['.js', '.jsx', '.ts', '.tsx', '.vue', '.svg', '.png'],
```

Comment:
Should the logo still be an image? What font-face is used? Perhaps we can add it in without embedding a png in the bundle.

Reply:
I don't know what actual font face was used for this logo, so I just use it as png image. @dustinbyrne